core/sdk-core/src/main/java/software/amazon/awssdk/core/internal/http/pipeline/stages/utils/RetryableStageHelper.java

```diff
@@ -140,14 +139,8 @@ public void logBackingOff(Duration backoffDelay) {
      * Retrieve the request to send to the service, including any detailed retry information headers.
      */
     public SdkHttpFullRequest requestToSend() {
-        Integer availableRetryCapacity = TokenBucketRetryCondition.getCapacityForExecution(context.executionAttributes())
-                                                                  .map(TokenBucketRetryCondition.Capacity::capacityRemaining)
-                                                                  .orElse(null);
-        String headerValue = (attemptNumber - 1) + "/" +
-                             context.executionAttributes().getAttribute(LAST_BACKOFF_DELAY_DURATION).toMillis() + "/" +
-                             (availableRetryCapacity != null ? availableRetryCapacity : "");
         return request.toBuilder()
-                      .putHeader(SDK_RETRY_INFO_HEADER, headerValue)
+                      .putHeader(SDK_RETRY_INFO_HEADER, "attempt=" + attemptNumber + "; max=" + retryPolicy.numRetries())
```

Comment:
I think the retry capacity is very valuable for troubleshooting; otherwise there's no way to tell if the retry gets throttled. I've seen an issue before where all requests failed of `IOException`, and I initially thought all retry attempts failed, but this header made me realized that the request did not get retried because there was no retry capacity.

Reply:
I guess now that we have metrics, we can see it there. It's just not as obvious

Reply:
Maybe we could add a debug log whenever we do not retry because of throttling?